libs/components/forms/src/lib/modules/input-box/input-box.component.ts

```diff
@@ -55,6 +71,53 @@ export class SkyInputBoxComponent implements OnInit {
   @Input()
   public disabled: boolean | undefined;
 
+  /**
+   * The text to display as the input's label and in known validation error messages. The label
+   * will automatically be associated with the `input`, `select`, or `textarea` element included
```

Comment:
should we tack 'sky ux input component' onto the list? Wondering if people will interpret that its only usable with plain ole inputs

Reply:
I added "compatible SKY UX component" to the list.